lib/fetch_votes.js

```diff
@@ -13,6 +14,19 @@ const fetchVotes = async () => {
   const votes = await voting.votes()
   console.debug(`fetchVotes`, `votes: ${votes}`)
 
+  const apps = await fetchApps()
+  const voteDescriptions = {}
+
+  // make concurrent requests for scripts
+  await Promise.all(votes.map(async vote => {
+    const describedScript = await describeScript(vote.script, apps)
```

Comment:
Why are apps required?

Reply:
I’m not quite sure, but it seems to be required by the Radspec interpreter.